tests/plugins/pastefromword/manual/listindentation.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: bug, 4.12.0, 2870, pastefromword
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, pastefromword, sourcearea, list
+
+Paste text from Word with indented lists. You can use example file: [List_indentation.docx](../generated/_fixtures/List_indentation/List_indentation.docx).
```

Comment:
I was thinking about this test and how to make it easier to digest, because now it's a little hard to understand to someone not familiar with this change. The only thing I came up with, it to modify the example document so list items describes what type of margin is used:

![image](https://user-images.githubusercontent.com/1061942/57531208-3a4d8600-7339-11e9-81c2-95eeef739be5.png)

So then when it is pasted you more or less know which items should have margins. Then the expected description would be that:

_Default and negative margins items should not have `margin-left` when pasted and positive ones should have `margin-left`_.

It could be also added as separate test, to have one generic (for people to experiment) and one more strict test. I'm not sure, do you think it could be useful?

Reply:
I'll update `.docx` file as you suggests, I don't have any better idea here too.

> It could be also added as separate test, to have one generic (for people to experiment) and one more strict test. I'm not sure, do you think it could be useful?

I'm for having one test file, it can be used for experimenting too.

Reply:
One more thing, have you thought about using default margins (`8px`) and not `0` for negative items? I'm not sure if there are any cases, so I guess it could wait until we get any feedback on this. Still it will make the list items more aligned in some cases 🤔

Reply:
>One more thing, have you thought about using default margins (8px) and not 0 for negative items?

I'd keep 0, because this is what we accept in editor.
I assume it is possible to adjust default list indentation to match the one in HTML, if not by config option, then by addon. If there is no such addon yet, than it can be done manually. In such case it would be expected that pasted list would look the same in editor as in Word.
If we change minimal value to `8px` for Word pasted content than you could create list in Word that would look exactly the same as list created via `list` plugin, however after pasting it to editor it would look different.